htdocs/fourn/facture/card.php

```diff
@@ -3237,6 +3237,17 @@ function setRadioForTypeOfInvoice() {
 			print '</span>';
 			print '</td>';
 			print '<td class="right'.($resteapayeraffiche ? ' amountremaintopay' : (' '.$cssforamountpaymentcomplete)).'">'.price($resteapayeraffiche).'</td>';
+
+			// Remainder to pay Multicurrency
+			if ($object->multicurrency_code != $conf->currency || $object->multicurrency_tx != 1) {
```

Comment:
Do you mean
if ($object->multicurrency_code != $conf->currency && $object->multicurrency_tx != 1) {
instead of
if ($object->multicurrency_code != $conf->currency || $object->multicurrency_tx != 1) {
?